src/main/webapp/i18n/de/guided-tour_course-overview.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
If we follow the current naming scheme of json translation files, the filename would be `guidedTourCourseOverview.json`